skimage/measure/tests/test_blur_effect.py

```diff
@@ -32,21 +34,37 @@ def test_blur_effect_channel_axis():
     """Test that passing an RGB image is equivalent to passing its grayscale
     version.
     """
-    image = astronaut()
-    B0 = blur_effect(image, channel_axis=-1)
-    B1 = blur_effect(rgb2gray(image))
-    B0_arr = blur_effect(image, channel_axis=-1, reduce_func=None)
-    B1_arr = blur_effect(rgb2gray(image), reduce_func=None)
+    image = ski.data.astronaut()
+    B0 = ski.measure.blur_effect(image, channel_axis=-1)
+    B1 = ski.measure.blur_effect(ski.color.rgb2gray(image))
+    B0_arr = ski.measure.blur_effect(image, channel_axis=-1, reduce_func=None)
+    B1_arr = ski.measure.blur_effect(ski.color.rgb2gray(image), reduce_func=None)
     assert 0 <= B0 < 1
     assert B0 == B1
-    assert_array_equal(B0_arr, B1_arr)
+    np.testing.assert_array_equal(B0_arr, B1_arr)
 
 
 def test_blur_effect_3d():
     """Test that the blur metric works on a 3D image."""
-    image_3d = cells3d()[:, 1, :, :]  # grab just the nuclei
-    B0 = blur_effect(image_3d)
-    B1 = blur_effect(gaussian(image_3d, sigma=1))
-    B2 = blur_effect(gaussian(image_3d, sigma=4))
+    image_3d = ski.data.cells3d()[:, 1, :, :]  # grab just the nuclei
+    B0 = ski.measure.blur_effect(image_3d)
+    B1 = ski.measure.blur_effect(ski.filters.gaussian(image_3d, sigma=1))
+    B2 = ski.measure.blur_effect(ski.filters.gaussian(image_3d, sigma=4))
     assert 0 <= B0 < 1
     assert B0 < B1 < B2
+
+
+@pytest.mark.parametrize('image', [np.zeros((100, 100, 3)), np.ones((100, 100, 3))])
+def test_blur_effect_uniform_input(image):
+    """Test that the blur metric is 1 for completely uniform images."""
+    B = ski.measure.blur_effect(image, channel_axis=-1)
```

Comment:
```suggestion
@pytest.mark.parametrize('factor', [0, 1, 2.5])
def test_blur_effect_uniform_input(factor):
    """Test that the blur metric is 1 for completely uniform images."""
    image = np.ones((10, 10, 3)) * factor
    B = ski.measure.blur_effect(image, channel_axis=-1)
```
Seems a bit more to the point and uses a bit less memory / resources.